application-analytics-ui/src/main/resources/Analytics/Code/Configuration.xml

```diff
@@ -0,0 +1,403 @@
+      <customClass/>
+      <customMapping/>
+      <defaultViewSheet/>
+      <defaultEditSheet/>
+      <defaultWeb/>
+      <nameField/>
+      <validationScript/>
+      <authToken>
+        <algorithm/>
+        <customDisplay/>
+        <disabled>0</disabled>
+        <hint/>
+        <name>authToken</name>
+        <number>3</number>
+        <picker>1</picker>
+        <prettyName>Authentication Token </prettyName>
+        <size>30</size>
+        <storageType>Hash</storageType>
+        <unmodifiable>0</unmodifiable>
+        <validationMessage/>
+        <validationRegExp/>
+        <classType>com.xpn.xwiki.objects.classes.PasswordClass</classType>
+      </authToken>
+      <idSite>
+        <customDisplay/>
+        <disabled>0</disabled>
+        <hint/>
+        <name>idSite</name>
+        <number>3</number>
+        <numberType>long</numberType>
+        <prettyName>ID Site</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <validationMessage/>
+        <validationRegExp/>
+        <classType>com.xpn.xwiki.objects.classes.NumberClass</classType>
+      </idSite>
+      <requestAddress>
+        <customDisplay/>
+        <disabled>0</disabled>
+        <hint/>
+        <name>requestAddress</name>
+        <number>4</number>
+        <picker>1</picker>
+        <prettyName>requestAddress</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <validationMessage/>
+        <validationRegExp/>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </requestAddress>
+      <trackerCode>
+        <contenttype>PureText</contenttype>
+        <customDisplay/>
+        <disabled>0</disabled>
+        <editor>PureText</editor>
+        <hint/>
+        <name>trackerCode</name>
+        <number>1</number>
+        <picker>1</picker>
+        <prettyName>Tracker Code</prettyName>
+        <rows>20</rows>
+        <size>40</size>
+        <unmodifiable>0</unmodifiable>
+        <validationMessage/>
+        <validationRegExp/>
+        <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
+      </trackerCode>
+    </class>
+    <property>
+      <authToken>DSFSDFFDSFSD</authToken>
+    </property>
+    <property>
+      <idSite/>
+    </property>
+    <property>
+      <requestAddress/>
+    </property>
+    <property>
+      <trackerCode>&lt;!-- Matomo --&gt;
+&lt;script&gt;
+  var _paq = window._paq = window._paq || [];
+  /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+  _paq.push(['trackPageView']);
+  _paq.push(['enableLinkTracking']);
+  (function() {
+    var u="http://130.61.233.19/matomo/";
+    _paq.push(['setTrackerUrl', u+'matomo.php']);
+    _paq.push(['setSiteId', '3']);
+    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+    g.async=true; g.src=u+'matomo.js'; s.parentNode.insertBefore(g,s);
+  })();
+&lt;/script&gt;
```

Comment:
Is this code taken from the matomo server? Is the admin configuring this matomo integration expected to add it himself from the matomo server, or it's the same for all users? I am asking since if this is not generic (it contains that `u`), we should not define it already on the configuration page